src/app/store/transactions/token-transfer.hooks.ts

```diff
@@ -121,9 +122,10 @@ export function useGenerateFtTokenTransferUnsignedTx(info: Sip10CryptoAssetInfo)
           standardPrincipalCVFromAddress(recipient),
         ];
 
-        if (info.hasMemo) {
-          functionArgs.push(memo);
-        }
+        // if info.hasMemo is false for DOGE so this doesn't get triggered
+        // if (info.hasMemo) {
```

Comment:
@kyranjamie @alter-eggo : This is a DRAFT that fixes SIP-10 send for some tokens like `DOGE` for example. 

In testing I saw that `info.hasMemo` is false for `DOGE` and so we don't push the memo in. 

I found that we have other code above to set the value to `noneCV()` if there is no `memo`.

It seems like we always need to push some value for `memo`, is that correct? 

I checked this refactor: https://github.com/leather-wallet/extension/pull/5414/files#r1609726172 and this check was just moved and not changed.

Reply:
That's my understanding as well, LGTM